code/dashboard-2/src/Constants.ts

```diff
@@ -721,3 +723,15 @@ export const JOB_QUERY_VALIDATION_SCHEMA = yup.object({
     .required('GPU usage is required')
 })
 
+export const JOB_QUERY_EXPORT_VALIDATION_SCHEMA = yup.object({
+  format: yup.string().required('A format must be selected'),
+  fields: yup.array().of(yup.string()).min(1, 'At least one field must be selected'),
+})
+
+export const JOB_QUERY_EXPORT_FORMATS: ExportFormat[] = [
+  {label: 'CSV with field header', value: 'csvnamed', mimeType: 'text/csv', fileExtension: 'csv'},
+  {label: 'CSV', value: 'csv', mimeType: 'text/csv', fileExtension: 'csv'},
```

Comment:
Should probably be 'CSV with header' since that is what you'll get.  Either that, or you find a way to add `noheader` to the format string.  Probably having a header is better in this case, as that format is most suitable for loading into spreadsheets and so on; people will use `awk` or `JSON` for things that will be machine-processed.